fabric-events-lifecycle-v0/src/main/java/net/fabricmc/fabric/api/event/server/ServerReloadCallback.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.event.server;
+
+import net.minecraft.server.MinecraftServer;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+/**
+ * A callback for the server's reloading.
```

Comment:
Whats being reloaded?

Reply:
`/reload` command execution. In 1.15, data pack reloading and block state cache recalculation.

Reply:
I should have been more clear, I was trying to hint at possibly explaing it as I dont think thats super clear. Sorry